pkgs/by-name/pl/play/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "play";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "paololazzari";
+    repo = "play";
+    tag = "v${version}";
+    sha256 = "sha256-31naTjYwCytytKXg9tQo2qx0hVoBwBwL7nVeoAV+/go=";
+  };
+
+  vendorHash = "sha256-9eP0rhsgpTttYrBG/BNk/ICtaM+zKNBz2H2cHuTSt30=";
+
+  modRoot = ".";
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = [ "version" ];
```

Comment:
```suggestion
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  versionCheckProgramArg = [ "version" ];
  doInstallCheck = true;
```

Reply:
That create issue.
Is because have remove doInstallCheck = true;

Reply:
What kind of issue? I just tested the build locally with `doInstallCheck = true;` and it worked fine. Without it the `versionCheckHook` would not get executed and would thus be useless here

Reply:
Have miss that.